### PR TITLE
Batch list editions

### PR DESCRIPTION
**Pull request by isthistechsupport**
I've added ways to batch edit parameter lists (including the trigger words and the channel ID blacklists and whitelists, when we add the whitelists lol)

### CODE REVIEW

**Comment by isthistechsupport** [comment]
Also did some refactoring and applied some DRY

File: bot.py [hunk 1]
```diff
@@ -159,9 +157,9 @@ def run_if_author_is_admin(interaction: discord.Interaction, function: Callable[
         return "You don't have permission to do that"
 
 
-def change_config_value(interaction: discord.Interaction, config_key: str, new_value: float) -> None:
+def change_config_value(interaction: discord.Interaction, config_key: str, new_value: Any) -> None:
```
Comment (Anlesvavor) [inline bot.py:160]: It has to be Any?

**Review by Anlesvavor — APPROVED** [review]
lgtm

File: bot.py (hunk 1 above)
Comment (isthistechsupport) [reply bot.py:160]: yeah because it's now used to update the chance thresholds (floats) and the bypass prefix (string)

Comment (isthistechsupport) [reply bot.py:160]: I should probably add some docstrings soon lol

**Review by gageway1 — APPROVED** [review]

**Pull request merged**